### PR TITLE
separate other upgrade routes

### PR DESCRIPTION
**Pull request by alexnaish**
- Move out the form handler and confirmation page routes
- Move views into their own folder for cleanliness

### CODE REVIEW

**Comment by alexnaish** [comment]
Same E2E test as before completed successfully (manual)

File: server/routes/upgrade/controller/confirmation.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import * as c from '../constants';
```
Comment (andygout) [inline server/routes/upgrade/controller/confirmation.js]: Maybe better to be verbose here: `import * as constants from '../constants';`. Single letters always feel a bit cryptic even when their use is obvious.

File: server/routes/upgrade/controller/confirmation.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+	flashMessage = JSON.parse(req.cookies[c.FLASH_COOKIE_NAME]);
+	res.clearCookie(c.FLASH_COOKIE_NAME, { path: '/upgrade' });
+	view.setSubscription(flashMessage.subscription);
+	view.model.offerId = flashMessage.offerId;
+
+	if (req.query.returnLocation !== undefined) {
+		view.model.returnLocation = `https://www.ft.com/${req.query.returnLocation}`;
+	}
+
+	SignupApi.fetchUserProfile({FTSecureSessionToken, userId, flags: res.locals.flags})
```
Comment (andygout) [inline server/routes/upgrade/controller/confirmation.js]: Spaces between curly braces for consistency with rest of code: `{ FTSecureSessionToken, userId, flags: res.locals.flags }`

File: server/routes/upgrade/controller/formHandler.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
+import * as c from '../constants';
```
Comment (andygout) [inline server/routes/upgrade/controller/formHandler.js]: As above.

File: server/routes/upgrade/controller/formHandler.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+	const upgradeRequestPayload = Object.assign({}, {
+		offerId: req.query.offerId,
+		type: 'Immediate',
+		useTestApi: res.locals.flags.nextSubscriptionAppWithTestApis,
+		subscriptionNumber: requestBody.subscriptionNumber,
+		term: requestBody.term,
+		countryCode: requestBody.countryCode
+	});
+	logger.info('operation=upgradeFormHandler action=transitionSubscription userId=${userId}');
+	SubscriptionTransitionApi.transitionSubscription(upgradeRequestPayload)
```
Comment (andygout) [inline server/routes/upgrade/controller/formHandler.js]: Spaces between above two lines for readability.

**Comment by alexnaish** [comment]
Updated as per comments and tested locally successfully

**Pull request merged**